test/new-e2e/tests/process/fargate_test.go

```diff
@@ -46,8 +46,13 @@ func getFargateProvisioner(configMap runner.ConfigMap) provisioners.TypedProvisi
 func TestECSFargateTestSuite(t *testing.T) {
 	t.Parallel()
 	s := ECSFargateSuite{}
+
+	extraConfig := runner.ConfigMap{
+		"ddagent:extraEnvVars": auto.ConfigValue{Value: "DD_PROCESS_CONFIG_RUN_IN_CORE_AGENT_ENABLED=false"},
+	}
```

Comment:
This test's intention is to validate the process checks running in the process-agent, but it broke after we defaulted the process checks to run in the core agent. This explicitly sets the check to run in the `process-agent` to ensure the data is coming from the expected agent.